modules/sdk-core/src/bitgo/pendingApproval/pendingApproval.ts

```diff
@@ -178,6 +178,11 @@ export class PendingApproval implements IPendingApproval {
       canRecreateTransaction = false;
     }
 
+    // If there are no recipients, then the transaction cannot be recreated
+    if (!!this.info()?.transactionRequest?.buildParams?.recipients) {
```

Comment:
I think this should be `if (!this.info()?.transactionRequest?.buildParams?.recipients?.length && this.info()?.transactionRequest?.buildParams?.type !== 'consolidate') {`

Or possibly nicer:
```
const recipients = this.info()?.transactionRequest?.buildParams?.recipients || [];
const type = this.info()?.transactionRequest?.buildParams?.type;
if (recipients.length === 0 && type !== 'consolidate') {
```